pre_commit_hooks/terraform/tflint.sh

```diff
@@ -2,6 +2,22 @@
 
 set -e
 
-for file in "$@"; do
-  tflint "$file"
-done
+if ! command -v tflint >/dev/null 2>&1; then
+  echo >&2 "tflint is not available on this system."
+  echo >&2 "Please visit https://github.com/terraform-linters/tflint for instructions on how to install."
+  exit 1
+fi
+
+tflint_version=$(tflint -v | awk 'NR==1 { print $3 }' | awk -F. '{ print $1"."$2 }')
+max_old_version="0.46"
+
+if [ "$(printf "%s\n" "$tflint_version" "$max_old_version" | sort -Vr | head -n 1)" = "$max_old_version" ]; then
```

Comment:
```suggestion
if [[ "$(printf "%s\n" "$tflint_version" "$max_old_version" | sort -Vr | head -n 1)" == "$max_old_version" ]]; then
```
best to use bash builtin as `test` can vary on OS's

Reply:
thinking about it, I would simplify by using awk to check the version since it handles numbers better, something like:
```
if [[ $(tflint -v | awk -vmaxv=$max_old_version 'NR==1 { version=$3; if(version > maxv){print "true"} }') == "true" ]]
```

Reply:
thx for suggestion